### PR TITLE
style: css scoping

### PR DESCRIPTION
**Pull request by MatthieuJnon**
Added a namespace to all css classes to prevent outside classes from affecting our css

### CODE REVIEW

File: src/scss/common.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
-@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import url('https://fonts.googleapis.com/css?family=Poppins');
 
-$roboto: 'Roboto', serif;  
+$roboto: 'Poppins', serif;
```
[the remark this answers is not in the document]
Comment (znat) [reply src/scss/common.scss]: Inter is a wise choice. Ok with me

[the remark this answers is not in the document]
Comment (znat) [reply src/scss/common.scss]: Regarding the height, I think the current one is too small. But your point is valid, so we could offre the option to hide the launcher when the chat is open (enabled with a prop). In that case a cross must be visible in the header (there's already a prop for the cross I think)

**Comment by GuillaumeTech** [comment]
I tried with rasa and quick reply and images messages, It was working fine, so nothing new to add to the review

**Comment by znat** [comment]
A good test would be to try it on our the website for example

**Pull request merged**